std/assembly/typedarray.ts

```diff
@@ -1027,6 +1159,53 @@ function FIND_INDEX<TArray extends ArrayBufferView, T>(
   return -1;
 }
 
+// @ts-ignore: decorator
+@inline
+function INCLUDES<TArray extends ArrayBufferView, T>(
+  array: TArray,
+  searchElement: T,
+  fromIndex: i32,
+): bool {
+  return INDEX_OF<TArray, T>(array, searchElement, fromIndex) >= 0;
+}
+
+// @ts-ignore: decorator
+@inline
+function INDEX_OF<TArray extends ArrayBufferView, T>(
+  array: TArray,
+  searchElement: T,
+  fromIndex: i32,
+): i32 {
+  var length = array.length;
+  if (length == 0 || fromIndex >= length) return -1;
+  if (fromIndex < 0) fromIndex = max(length + fromIndex, 0);
+  var dataStart = array.dataStart;
+  while (fromIndex < length) {
+    if (load<T>(dataStart + (<usize>fromIndex << alignof<T>())) == searchElement) return fromIndex;
```

Comment:
Remembers me of the previous usize discussion, with the difference that pretty much all code in stdlib does it exactly like done here. That is: Extending i32->i64 in a loop instead of wrapping i64->i32 exactly once on return. If WASM64 ever becomes a thing, we'll most likely want to do micro-optimize all of these to do it the other way around.

Reply:
Done!